infra/src/main/kotlin/fi/fta/geoviite/infra/linking/PublishDao.kt

```diff
@@ -79,7 +79,11 @@ class PublishDao(jdbcTemplateParam: NamedParameterJdbcTemplate?) : DaoBase(jdbcT
 
     fun fetchSwitchPublishCandidates(): List<SwitchPublishCandidate> {
         val sql = """
-            select coalesce(draft_of_switch_id, id) as official_id, name, change_time
+            select coalesce(draft_of_switch_id, id) as official_id, name, change_time,
```

Comment:
Nää muutokset tulee conflictaamaan aika raskaasti niiden juttujen kanssa mitä oon itse tehnyt, kun oon kanssa säätänyt näitä SQL-lauseita 😅 Täytyy olla mergeissä varovainen sitten kun tehdään